pkg/api/podsecuritypolicy/util.go

```diff
@@ -38,3 +35,16 @@ func DropDisabledFields(pspSpec, oldPSPSpec *policy.PodSecurityPolicySpec) {
 		}
 	}
 }
+
+func allowedProcMountTypesInUse(oldPSPSpec *policy.PodSecurityPolicySpec) bool {
```

Comment:
Can we use AllowedProcMountTypes as the name of the function to relate it with the  function of util_test.go?

Reply:
this is fine as is, the field is AllowedProcMountTypes, this function is checking if it is in use, and we want it private to the package

Reply:
Thanks for your explanation, it makes me clearer. I saw it from another PR.